web/ui/react-app/src/QueryStatsView.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './QueryStatsView.css';
+
+export interface QueryStats {
+    loadTime: number;
+    resolution: number;
+    totalTimeSeries: number;
+}
+
+const QueryStatsView = (props: QueryStats) => {
+    const {loadTime, resolution, totalTimeSeries} = props;
+
+    return (
+        <div className="query-stats">
+            <span className="float-right">Load Time: {loadTime}ms &ensp; Resolution: {resolution} &ensp; Total Time Series: {totalTimeSeries}</span>
```

Comment:
Here's where my specific sense of aesthetics kicks in ;)

Could we change this to:

```suggestion
            <span className="float-right">Load time: {loadTime}ms &ensp; Resolution: {resolution}s &ensp; Result series: {totalTimeSeries}</span>
```

(shorter wording, no title case, and adding a unit to the resolution)

Reply:
🤣 I looked at it and thought about the case, but went with this one. Will change it.